mobile-first.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bezalell Portfolio</title>
+  </head>
+  <body>
+    <!-- navbar -->
+    <header>
+      <nav class="navbar">
+        <div class="name-container">
+          <a href="index.html">
+            <h2>Bezalell</h2>
+          </a>
+        </div>
+        <div>
+          <div class="bar1"></div>
+          <div class="bar2"></div>
+          <div class="bar3"></div>
+        </div>
+      </div>
+      <div class="links-container">
+        <ul>
+          <li>
+            <a>Portfolio</a>
+          </li>
+          <li>
+            <a>About</a>
+          </li>
+          <li>
+            <a>Contact</a>
+          </li>
+          <li>
+            <a
+            ><img
+            src="images/Normal Button/Tertiary/Disabled.png"
+            alt="Message"
+            /></a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <!-- header -->
+    <section class="header">
```

Comment:
1. Overall, you design elements are aligned to the left. While in the Figma design, the elements should not be squeezed towards the left sized. This could be resolved by giving a total of `24px` to both right and left sides. And then given the recommended dimensions and `font-size` to elements/text.
2. The space between the social media icons and the end of the page section in your design is very small as compared to the one in Figma. Kindly try to fix the spacing.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/220970496-5a66397d-e934-4952-94f0-7d352d9d14c4.png)| ![image](https://user-images.githubusercontent.com/61361037/220969582-50d961bc-7285-419c-87db-c3449c52618b.png)|